source/common/router/router.h

```diff
@@ -648,20 +643,26 @@ class Filter : Logger::Loggable<Logger::Id::router>,
   // list of cookies to add to upstream headers
   std::vector<std::string> downstream_set_cookies_;
 
+  Network::TransportSocketOptionsConstSharedPtr transport_socket_options_;
+  Network::Socket::OptionsSharedPtr upstream_options_;
+  // Set of ongoing shadow streams which have not yet received end stream.
+  absl::flat_hash_set<Http::AsyncClient::OngoingRequest*> shadow_streams_;
+
+  // Keep small members (bools and enums) at the end of class, to reduce alignment overhead.
+  uint32_t retry_shadow_buffer_limit_{std::numeric_limits<uint32_t>::max()};
+  uint32_t attempt_count_{1};
+  uint32_t pending_retries_{0};
+  Http::Code timeout_response_code_ = Http::Code::GatewayTimeout;
+  FilterUtility::HedgingParams hedging_params_;
```

Comment:
Might also be good to add `: 1` to [hedge_on_per_try_timeout_](https://github.com/envoyproxy/envoy/blob/f5677953923766d9b101c798de3896d3504a60f4/source/common/router/router.h#L65)

Reply:
Done